cmake/Modules/LocalAom.cmake

```diff
@@ -23,6 +23,27 @@ if(EXISTS "${LIB_FILENAME}")
         # ext/avm/aom/aom_encoder.h includes config/aom_config.h which is generated by the local build of avm.
         target_include_directories(aom INTERFACE "${AOM_EXT_SOURCE_DIR}/build.libavif")
     endif()
+
+    # Add link dependency flags from the aom.pc file in ext/aom or ext/avm
+    # by prepending the build directory to PKG_CONFIG_PATH and then calling
+    # pkg_check_modules
+    if(WIN32)
+        set(ENV{PKG_CONFIG_PATH} "${AOM_EXT_SOURCE_DIR}/build.libavif;$ENV{PKG_CONFIG_PATH}")
+    else()
+        set(ENV{PKG_CONFIG_PATH} "${AOM_EXT_SOURCE_DIR}/build.libavif:$ENV{PKG_CONFIG_PATH}")
+    endif()
+
+    pkg_check_modules(_AOM QUIET aom)
```

Comment:
Did you test this part?

CMake documentation on [Set Environment Variable](https://cmake.org/cmake/help/latest/command/set.html#set-environment-variable) says:

> This command affects only the current CMake process, not the process from which CMake was called, nor the system environment at large, nor the environment of subsequent build or test processes.

This seems to imply that if `pkg_check_modules()` executes the `pkg-config` command, the `pkg-config` process (which I think is a "subsequent build process") won't see the new value of the `PKG_CONFIG_PATH` environment variable.

Reply:
I did test this part. I just double checked by adding `-lfoo` to ext/aom/build.libavif/aom.pc `Libs`. It looks like environment variables set from cmake files get passed along in calls to `execute_process`.

Reply:
Thank you for the confirmation and thank you for double checking. I misunderstood what "subsequent build or test processes" means. I think it means a subsequent invocation of a build or test command, rather than the a child process of the current `cmake` process.